### ISSUE TITLE
Insufficient gem dependency for numo/narray

### ISSUE THREAD

**Issue by k0kubun** [red-data-tools/charty#51]
```
source 'https://rubygems.org'
gem 'charty', '0.2.1'
```

```
$ ruby -v
ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-linux]
$ bundle exec ruby -e "require 'charty'"
Traceback (most recent call last):
        5: from -e:1:in `<main>'
        4: from -e:1:in `require'
        3: from /home/k0kubun/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/charty-0.2.1/lib/charty.rb:12:in `<top (required)>'
        2: from /home/k0kubun/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/charty-0.2.1/lib/charty.rb:12:in `require_relative'
        1: from /home/k0kubun/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/charty-0.2.1/lib/charty/palette.rb:1:in `<top (required)>'
/home/k0kubun/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/charty-0.2.1/lib/charty/palette.rb:1:in `require': cannot load such file -- numo/narray (LoadError)
```

Is this intentional?

**Comment by mrkn** [comment]
It is intentional that we don't put numo-narray in the runtime dependencies because we want to support JRuby completely in the future.
But, it isn't intentional that numo-narray is loaded just when charty is loaded.
I'll fix it.

**Comment by mrkn** [comment]
@k0kubun I've released version 0.2.3.  This version doesn't require numo-narray.
